frontend/src/main/components/Users/UsersTable.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import OurTable from "main/components/OurTable"
+import OurTable, { ButtonColumn } from "main/components/OurTable"
```

Comment:
I suggest that you do not try to combine this change in the same PR as changing the entity to also include the driver.  It isn't necessary, and it complicates things.